lib/pacemaker/pcmk_sched_location.c

```diff
@@ -128,13 +129,18 @@ generate_location_rule(pcmk_resource_t *rsc, xmlNode *rule_xml,
 
     rule_xml = pcmk__xe_resolve_idref(rule_xml, rsc->priv->scheduler->input);
     if (rule_xml == NULL) {
-        return false; // Error already logged
+        pcmk__config_err("Ignoring location constraint '%s' because it "
```

Comment:
could use neither/nor

Reply:
The `pcmk__xe_resolve_idref()` message is sufficient for that case, nothing needed here

Reply:
I added it because otherwise nothing says we're ignoring the constraint. But that's probably fine..

Reply:
(since we dropped the message from `unpack_rsc_location()` just now

Reply:
alright yeah if you don't want that, pushing an update without it

Reply:
Well, it will say it's ignoring the invalid rule configuration, which is close enough

Reply:
(compared to logging two messages)